drivers/lte_link_control/Kconfig

```diff
@@ -83,6 +83,19 @@ config LTE_PDP_CONTEXT
 		The +CGDCONT command defines Packet Data Protocol (PDP) Context.
 		For reference, see 3GPP 27.007 Ch. 10.1.1
 
+choice
+	prompt "Select network mode"
+	default LTE_NETWORK_MODE_LTE
+	help
+		Select either LTE or Narrowband-IOT (NB_IOT) network mode
+
+config LTE_NETWORK_MODE_LTE
+	bool "LTE"
```

Comment:
Perhaps this could rather be "LTE_M" or "CAT_M1" to be even more specific? NB-IoT is sometimes referred to as LTE Cat NB1 so could be a bit confusing.